apps/labs/posts/scipy-array-api.md

```diff
@@ -0,0 +1,441 @@
+for distributed and GPU arrays' on its roadmap" for five years now.
+See [this page](https://data-apis.org/array-api/latest/use_cases.html#use-case-scipy) on the use cases of the
+array API and [this blog post](https://labs.quansight.org/blog/2021/11/pydata-extensibility-vision) by
+Ivan Yashchuk and Ralf Gommers for more information.
+
+Due to these performance wishes, many different "array provider" libraries, such as CuPy and
+PyTorch, have been developed, with support for one or more of these hardware use-cases in mind.
+However, due to the fact that "array consumer" libraries are typically built to be compatible with
+only one array provider library, many functions have been reimplemented in different consumer libraries,
+just to work with a different provider library.
+In extreme cases, this has led to efforts to reimplement entire consumer libraries,
+for example [`cupyx.scipy`](https://docs.cupy.dev/en/stable/reference/scipy.html).
+This constant reimplementation of the same functionality has led many to see the Scientific Python Ecosystem
+as rather being fractured into many smaller ecosystems, one built around each array library, and to dream for a
+world where the ecosystems are unified and everything works together!
+
+While it would certainly be a very ambitious goal to aim for _everything_ working together, the goal
+of increasing interoperability between array libraries is one with widespread support.
+There are currently many hurdles which users can run into when navigating this fractured ecosystem.
+They can find that they have to convert between different array provider libraries in their projects, in order to
+make use of the full range of tools which have been developed.
+For users who decide to use a different provider library to take advantage of performance benefits, they can
+find themselves needing to learn new APIs to accomplish the same tasks, which are often similar but frustratingly
+different to those which they are used to.
+For large existing projects, this makes the prospect of switching to a different provider library a very daunting one.
+
+On the developer side of things, it seems like there is a lot of wasted time in reimplementing the same functionality
+for many different provider libraries.
+If each piece of functionality only needed to be implemented once and worked with any array library, then there would 
+be many fewer projects to maintain, and much more time to work on implementing brand new functionality.
+Fortunately, there has been work on a standard which will help push us towards this imagined future.
+
+---
+
+## The Array API Standard
+
+[The Consortium for Python Data API Standards](https://data-apis.org/) has created
+[the Python array API standard](https://data-apis.org/array-api/), which is a "specification for what it means for a
+library to have an ‘array API’" (see [Aaron's talk](https://youtu.be/16rB-fosAWw)).
```

Comment:
One could consider adding a line or two on the history of the Array API standard. Others know more, but I think it was years in the making and conversation between many organisations.

Reply:
I've added "The Consortium was formed in 2020 by a group of maintainers and industry stakeholders - see [the paper accompanying Aaron's talk](https://conference.scipy.org/proceedings/scipy2023/pdfs/aaron_meurer.pdf) for more information." in https://github.com/Quansight/Quansight-website/pull/770/commits/c359314a43da5b53526b6c237d02e1251c4f9eac.